client/lib/signup/step-actions/index.js

```diff
@@ -539,6 +556,7 @@ export function addAddOnsToCart(
 		slug,
 		null,
 		null,
+		null,
```

Comment:
[nit] There are so many `null` 😂

Reply:
I've updated the function `processItemCart` to accept an object instead of individual arguments. LMK if that works better in your opinion!